src/DependencyInjection/BabDevPagerfantaExtension.php

```diff
@@ -51,6 +54,16 @@ protected function loadInternal(array $mergedConfig, ContainerBuilder $container
 
         if (interface_exists(NormalizerInterface::class)) {
             $loader->load('serializer.php');
+
+            if (class_exists(InstalledVersions::class)) {
+                $version = InstalledVersions::getVersion('symfony/serializer');
```

Comment:
Turns out, in order to properly use this, we need to depend on `composer-runtime-api ^2.0`. Then we can also remove the `class_exists(InstalledVersions::class)` check.

@mbabker Is it worth changing it?

Reply:
Unless it's creating a fatal error, I really don't think it's worth that much more extra effort.  For the Symfony 5.4 folks still using Composer 1.x, that setup is deprecated anyway as Symfony 6 requires Composer 2 and its runtime API.

At this point we're kind of bending over backwards to avoid the `DebugClassLoader` spitting out a dev-only message about implementing a deprecated interface, and from the PRs I've looked at across the ecosystem, nobody else looks to be trying to avoid that so far.